### PR TITLE
feat(cli): support .sunodo.env for sunodo run

### PR DESCRIPTION
**Pull request by endersonmaia**
This PR will make `sunodo run` accept a `.sunodo.env` file which make the sunodo/rollups-node container load the environment variables, so that one can pass variables to the sunodo/rollups-node services.

### CODE REVIEW

File: apps/cli/src/commands/run.ts [hunk 1]
```diff
@@ -108,6 +108,11 @@ export default class Run extends Command {
         ];
         const attachment = flags.verbose ? [] : ["--attach", "validator"];
 
+        // create en empty .sunodo.env file if it doesn't exists
+        if (!fs.existsSync(".sunodo.env")) {
+            fs.writeFileSync(".sunodo.env", "");
```
Comment (tuler) [inline apps/cli/src/commands/run.ts:113]: research a way to not need this

File: apps/docs/guide/running/running-application.md [hunk 2]
```diff
@@ -16,6 +16,21 @@ The private chain by default has a block time of 5 seconds, controlled by `--blo
...
+All Rollups Node services can be configured using environment variables.
+
+For ex., in case you want to change the default deadline for adanving the state for the rollups-advance-runner service.
+
+```shell
+cat .sunodo.env
+SM_DEADLINE_ADVANCE_STATE=360000
+```
+
+Check the configuration options for each service in the [Rollups Node documentation](https://github.com/cartesi/rollups/blob/v1.0.1/offchain/README.md).
```
Comment (tuler) [inline apps/docs/guide/running/running-application.md:32]: ps: except that it's not documented there

File: apps/cli/src/commands/run.ts (hunk 1 above)
Comment (endersonmaia) [reply apps/cli/src/commands/run.ts:113]: already did, `compose` fails if the file doesn't exists

what I can do is create a `docker-compose-envfile.yaml` that would be added to `compose --file` only if the file exists.

File: apps/docs/guide/running/running-application.md (hunk 2 above)
Comment (endersonmaia) [reply apps/docs/guide/running/running-application.md:32]: yeah, what do you suggest?

I thought on instructing the user to nu `docker run ... cartesi/rollups-advance-runner --help` but I felt it's out of scope for this documentation, and ugly.

File: apps/cli/src/node/docker-compose-dev.yml [hunk 3]
```diff
@@ -90,6 +90,8 @@ services:
             retries: 5
         ports:
             - 8080:8080
+        env_file:
+            - ./.sunodo.env
```
Comment (tuler) [inline apps/cli/src/node/docker-compose-dev.yml:94]: Does this should be here? I see you are adding a compose file if the env file exists.

Comment (endersonmaia) [reply apps/cli/src/node/docker-compose-dev.yml]: Fixed!

**Review by tuler — APPROVED** [review]

**Pull request merged**